plone/formwidget/geolocation/profiles/default/registry.xml

```diff
@@ -12,13 +12,14 @@
       interface="Products.CMFPlone.interfaces.IBundleRegistry"
       prefix="plone.bundles/geolocation-bundle"
       purge="False">
+    <value key="merge_with">default</value>
     <value key="resources">
       <element>geolocation-bundle-resource</element>
     </value>
     <value key="jscompilation">++plone++plone.formwidget.geolocation/geolocation-bundle-compiled.js</value>
     <value key="csscompilation">++plone++plone.formwidget.geolocation/geolocation-bundle-compiled.css</value>
     <value key="last_compilation">2016-06-16 00:00:00</value>
-    <value key="depends">plone</value>
+    <value key="depends">patterns</value>
```

Comment:
This bundle does not necessarily depend on the patterns bundle. The plone bundle should include all the patternslib dependencies, which are needed. It's only necessary, that wether
- plone.patternslib is installed (so that the Plone cores patternslib resources are updated)
- Or this PR is merged: https://github.com/plone/Products.CMFPlone/pull/1644 (which it probably will, after the robot test is fixed)